Added header functionality

claudioLopezSA opened the pull request.

claudioLopezSA committed:
[Desktop]
Removed unused values

claudioLopezSA committed:
Merge branch 'feature/frontend' into feature/post

claudioLopezSA committed:
[Desktop]
Added some style values UserProfile.css
Added style values to UserProfileHeader.css
Added UserProfileModal.js
(UserProfileHeader.js)
Added div containers
Added state to handle modal
Added button for the modal

claudioLopezSA committed:
[Desktop]
Moved the content of modal to variable FooterReplyModal.js
Changed reactions to data httpService.js
(Post.js)
Removed unused code
Changed Toggle to ToggleCollapseComment handler
(PostFooter.js)
Added onHover func
Changed some style values
Removed unused array actionSpeedDialNav
Added BadgeCounterValue
Fixed comment button
Removed unused code

Loui-Molina committed:
[Desktop]
User Profile restructure

claudioLopezSA committed:
Merge remote-tracking branch 'origin/feature/frontend' into feature/post

# Conflicts:
#	DesktopReact/frontend/src/api/httpService.js

claudioLopezSA committed:
[Desktop]
Changed values of array httpService.js

claudioLopezSA committed:
[Desktop]
Added state to reactions

claudioLopezSA committed:
[Desktop]
(Fix) Refactor folder name

claudioLopezSA committed:
[Desktop]
Added style to component

claudioLopezSA committed:
[Desktop]
Updated complement
Added state for modal

claudioLopezSA committed:
[Desktop]
Added style to modal

claudioLopezSA committed:
[Desktop]
Added modal component

Loui-Molina reviewed (changes requested):
Little changes to do

claudioLopezSA committed:
Merge remote-tracking branch 'origin/feature/frontend' into feature/post

claudioLopezSA committed:
[Desktop]
Added functionality

claudioLopezSA committed:
[Desktop]
Updated onClick with handler

claudioLopezSA committed:
[Desktop]
Added FooterReplyModal
Added state to modal
Added stopPropagation
Removed unused code

claudioLopezSA committed:
[Desktop]
Updated loadProfiles func

claudioLopezSA committed:
[Desktop]
Added todo

claudioLopezSA committed:
[Desktop]
Removed unused code
Added todo

claudioLopezSA committed:
[Desktop]
Removed unused Typography
